src/test/java/org/apache/commons/collections4/FluentIterableTest.java

```diff
@@ -416,12 +416,6 @@ public void isEmpty() {
 
     @Test
     public void size() {
-        try {
-            FluentIterable.of((Iterable<?>) null).size();
-            fail("expecting NullPointerException");
-        } catch (final NullPointerException npe) {
-            // expected
-        }
```

Comment:
This test was covering the case of `FluentIterable.of(null)`, but there are other tests covering it too. And as pointed by the OP, this method was - according to its name - supposed to test the `.size()` method. :+1: